client/settings/__tests__/google-play-test-mode-compatibility-notice.test.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import GooglePayTestModeCompatibilityNotice from '../google-play-test-mode-compatibility-notice';
```

Comment:
```suggestion
import GooglePayTestModeCompatibilityNotice from '../google-pay-test-mode-compatibility-notice';
```
Ooh, this one is a bit more critical. Could we just update this filename and related imports too please? Ta. 👍

Reply:
Ugh, I thought I had caught everything in the last reivew. But no... she still plays when she should pay. My keyboard was never the faithful kind 💔
Appreciate you catching that, thank you for making me see the truth, sorry about the multiple review rounds! Time for me to move on...